frontend/packages/ceph-storage-plugin/src/components/modals/advanced-kms-modal/advanced-vault-modal.tsx

```diff
@@ -126,7 +131,7 @@ export const AdvancedVaultModal = withHandlePromise((props: AdvancedKMSModalProp
 
   return (
     <Form onSubmit={submit} key="advanced-vault-modal">
-      <div className="modal-content modal-content--no-inner-scroll">
```

Comment:
Was above class causing some issue ?? Asking because, if so, we need to remove it from `advanced-ibm-kms-modal.tsx` as well...

Reply:
no need to remove this on advanced-ibm-kms-modal.tsx, I just removed "--no-inner-scroll" from here. For vault SA, modal has some additional fields and it increased modal height, I removed this to enable scroll only for SA.